junction/conferences/migrations/0010_auto_20150713_2331.py

```diff
@@ -16,36 +16,100 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='ConferenceVenue',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='Created At')),
-                ('modified_at', models.DateTimeField(auto_now=True, verbose_name='Last Modified At')),
+                (
+                    'id',
+                    models.AutoField(
+                        verbose_name='ID',
+                        serialize=False,
+                        auto_created=True,
+                        primary_key=True,
+                    ),
+                ),
+                (
+                    'created_at',
+                    models.DateTimeField(auto_now_add=True, verbose_name='Created At'),
+                ),
+                (
+                    'modified_at',
+                    models.DateTimeField(
+                        auto_now=True, verbose_name='Last Modified At'
+                    ),
+                ),
                 ('name', models.CharField(max_length=100)),
                 ('address', models.TextField()),
                 ('latitude', models.DecimalField(max_digits=17, decimal_places=15)),
                 ('longitudes', models.DecimalField(max_digits=19, decimal_places=16)),
-                ('created_by', models.ForeignKey(related_name='created_conferencevenue_set', verbose_name='Created By', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
-                ('modified_by', models.ForeignKey(related_name='updated_conferencevenue_set', verbose_name='Modified By', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
+                (
+                    'created_by',
+                    models.ForeignKey(
+                        related_name='created_conferencevenue_set',
+                        verbose_name='Created By',
+                        blank=True,
+                        to=settings.AUTH_USER_MODEL,
+                        null=True,
+                    ),
+                ),
+                (
+                    'modified_by',
+                    models.ForeignKey(
+                        related_name='updated_conferencevenue_set',
+                        verbose_name='Modified By',
+                        blank=True,
+                        to=settings.AUTH_USER_MODEL,
+                        null=True,
+                    ),
+                ),
             ],
-            options={
-                'abstract': False,
-            },
+            options={'abstract': False,},
```

Comment:
See this for curly braces a well. Is there a line length till which using braces (or brackets) in single line is valid ?